public/userview.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="en">
+    <form action="http://localhost:8765" method="post">
```

Comment:
Doing it like this means the user will see just a class after submitting the file. Maybe it is better to do it with an AJAX request and show the server response somewhere.

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API